project_event/tests/test_security.py

```diff
@@ -659,3 +660,48 @@ def test_880_project_manager_can_delete_resource_calendar_instrument(self):
         self.assertTrue(
             self.Instruments.sudo(self.user_manager.id).search([]).unlink()
         )
+
+    def test_890_project_user_cannot_read_calendar_event(self):
+        self.get_user_acls_and_rules_to_model(self.project_user, self.Events)
+        with self.assertRaises(exceptions.AccessError):
+            self.Events.sudo(self.project_user.id).search([])
+
+    def test_900_project_user_cannot_create_calendar_event(self):
+        with self.assertRaises(exceptions.AccessError):
+            self.Events.sudo(self.project_user.id).create({})
+
+    def test_910_project_user_cannot_delete_calendar_event(self):
+        with self.assertRaises(exceptions.AccessError):
+            self.Events.sudo(self.project_user.id).search([]).unlink()
+
+    def test_920_user_editor_can_read_calendar_event(self):
```

Comment:
After this, you are testing for 'can' but you are waiting for an error, so testes 920 to 980 are not right